### PR TITLE
[SR-1606] [Commands] Fix warnings in template package.

### PR DESCRIPTION
**Pull request by ddunbar**
- This also adds more explicit variables to distinguish between where the
   example package, module, and type names are used.

 - Fixes: https://bugs.swift.org/browse/SR-1606

### CODE REVIEW

**Comment by ddunbar** [comment]
This is a PR to the preview branch.

File: Sources/Commands/init.swift [hunk 1]
```diff
@@ -30,10 +30,25 @@ extension InitError: CustomStringConvertible {
...
 
+/// Create an initial template package.
 final class InitPackage {
+    let rootd = POSIX.getcwd()
+
+    /// The mode in use.
     let mode: InitMode
+
+    /// The name of the example package to create.
     let pkgname: String
```
Comment (abertelrud) [inline Sources/Commands/init.swift]: This might be too picky to even mention, but shouldn't this use camel case like the other properties?  i.e. shouldn't it be "packageName"?

Comment (ddunbar) [inline Sources/Commands/init.swift]: Yes, it should and I thought the same thing, I just left it out to avoid unnecessary changes in the diff.

Comment (abertelrud) [inline Sources/Commands/init.swift]: Makes a lot of sense to minimize the diff.  We can always clean this up later.  It was indeed a nit-pick.
